src/oc/mswinprt.c

```diff
@@ -348,7 +348,7 @@ void plprint(const char* s) {
 	printf("%s", s);
 }
 #endif
-int hoc_plttext;
+//int hoc_plttext; //defined in plot.c
```

Comment:
@nrnhines : `hoc_plttext` is not appeared / used in this mswinprt.c file. Shouldn't we just remove this commented line as well?